Assets/MixedRealityToolkit/Providers/UnityInput/MouseDeviceManager.cs

```diff
@@ -62,21 +107,20 @@ public override void Enable()
             System.Type controllerType = typeof(MouseController);
 
             // Make sure that the handedness declared in the controller attribute matches what we expect
+            var controllerAttribute = MixedRealityControllerAttribute.Find(controllerType);
+            if (controllerAttribute != null)
             {
-                var controllerAttribute = MixedRealityControllerAttribute.Find(controllerType);
-                if (controllerAttribute != null)
-                {
-                    Handedness[] handednesses = controllerAttribute.SupportedHandedness;
-                    Debug.Assert(handednesses.Length == 1 && handednesses[0] == Handedness.Any, "Unexpected mouse handedness declared in MixedRealityControllerAttribute");
-                }
+                Handedness[] handednesses = controllerAttribute.SupportedHandedness;
+                Debug.Assert(
+                    handednesses.Length == 1 && handednesses[0] == Handedness.Any, 
+                    "Unexpected mouse handedness declared in MixedRealityControllerAttribute");
             }
 
-            IMixedRealityInputSystem inputSystem = Service as IMixedRealityInputSystem;
-
-            if (inputSystem != null)
+            // todo: this looks like it could be cleaned up / improved
```

Comment:
>// todo: this looks like it could be cleaned up / improved [](start = 12, length = 58)

Please get rid of TODOs or have an issue tracking them.

Generally speaking I haven't seen TODOs ever actually get fixed and context associated with them gets lost.

Reply:
missed that one. thanks!